_posts/2019-10-10-event-store-packages-for-amazon-linux-2.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: blog-post
+title: Event Store packages for Amazon Linux 2
+author: Arwin Neil Baichoo
+---
+# Event Store packages for Amazon Linux 2
+
+Event Store Commercial package for Amazon Linux 2 AMI is now available!
+
+#### Changes moving from Amazon Linux 1 to Amazon Linux 2 Packages
+---
+
+#### Phasing out support for AWS Auto Scaling groups 
+Running Event Store as in AWS EC2 Auto Scaling groups is no more a recommended practice because AGS can end up simultaneously or near simultaneously deleting all instances
+
+To learn more about running Event Store clusters, click [here](https://eventstore.org/docs/server/cluster-with-manager-nodes/index.html).
+
+#### Systemd instead of System V init system
+As of Amazon Linux 2, [System V init has been replaced with systemd service and systems manager](https://aws.amazon.com/amazon-linux-2/release-notes/). Event Store is now started using `systemctl` instead of `initctl` 

```

Comment:
```suggestion
As of Amazon Linux version 2, [System V init has been replaced with systemd service and systems manager](https://aws.amazon.com/amazon-linux-2/release-notes/). Event Store is now started using `systemctl` instead of `initctl` 
```